cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/adapter/AdapterService.java

```diff
@@ -374,10 +374,10 @@ private void addSchedule(Id.Program programId, SchedulableProgramType programTyp
     Preconditions.checkArgument(frequency != null,
                                 "Frequency of running the adapter is missing from adapter properties." +
                                   " Cannot schedule program.");
-    String cronExpr = Schedules.toCronExpr(frequency);
+    String cronExpr = SchedulesUtil.toCronExpr(frequency);
```

Comment:
actually, this could be a useful utility for developers. Why not have in Schedules?